core/error.rs

```diff
@@ -167,7 +167,7 @@ pub fn to_v8_error<'a>(
 pub(crate) fn call_site_evals_key<'a>(
   scope: &mut v8::HandleScope<'a>,
 ) -> v8::Local<'a, v8::Private> {
-  let name = v8_static_strings::CALL_SITE_EVALS.v8_string(scope);
+  let name = v8_static_strings::CALL_SITE_EVALS.v8_string(scope).unwrap();
```

Comment:
This unwrap has been pushed up to the caller. There's probably some questionable places we do this, but I didn't inspect them thoroughly because it's existing behaviour.

Reply:
Yeah, looks good